modules/exchainAnalyticsAdapter.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * This module adds UUID to bid
+ * @module modules/exchainAnalyticsAdapter
+ */
+
+import { getGlobal } from '../src/prebidGlobal.js';
+import adapterManager from '../src/adapterManager.js';
+import { getStorageManager } from '../src/storageManager.js';
+import { MODULE_TYPE_ANALYTICS } from '../src/activities/modules.js';
+
+export const MODULE_NAME = 'ExchainAnalyticsAdapter';
+const EXCHAIN_PREBID_GVL_ID = 6969
```

Comment:
Did you make this up

Reply:
I didn't find any documentation about logic of getting this ID. Where I can find it/

Reply:
well this module appears to be vendorless #11536 ?